### PR TITLE
Ethereum init

### PR DESCRIPTION
**Pull request by sagivo**
this PR will bring the ethereum lib to use our smart contracts!

### CODE REVIEW

File: ipfs/ipfsService.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ const get = async (key) => {
 }
 
 const stop = () => {
-  ipfs.stop(process.exit);
+  ipfs.stop();
```
Comment (feulf) [inline ipfs/ipfsService.js]: is this the bug the IPFS team fixed?

File: storage/ethereum.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+const Web3 = require('web3');
+const TruffleContract = require('truffle-contract');
+const storeSol = require('../build/contracts/Store.json');
+
+const web3 = new Web3(new Web3.providers.HttpProvider('http://localhost:9545'));
```
Comment (feulf) [inline storage/ethereum.js]: Can we put this into a config object so we can change it by the environment

Comment (sagivo) [reply storage/ethereum.js]: yes, we should. i will create a separate file for web3.

File: ipfs/ipfsService.js (hunk 1 above)
Comment (sagivo) [reply ipfs/ipfsService.js]: we spoke about the bug in ipfs, that's the fix

**Pull request merged**